pytorch_lightning/loggers/csv_logs.py

```diff
@@ -144,8 +144,8 @@ def __init__(
     def root_dir(self) -> str:
         """Parent directory for all checkpoint subdirectories.
 
-        If the experiment name parameter is ``None`` or the empty string, no experiment subdirectory is used and the
-        checkpoint will be saved in "save_dir/version_dir"
+        If the experiment name parameter is the empty string, no experiment subdirectory is used and the checkpoint will
+        be saved in "save_dir/version_dir"
         """
         if not self.name:
             return self.save_dir
```

Comment:
Technically we don't need these lines, and everything works the same since if you do `os.path.join(self._save_dir, self.name)` and `self.name` is an empty string, you will still get `self._save_dir`

Reply:
Do people see value in keeping these lines?

Reply:
nope!! as you suggested, this can be updated.